browser/net/brave_site_hacks_network_delegate_helper_unittest.cc

```diff
@@ -251,6 +252,7 @@ TEST(BraveSiteHacksNetworkDelegateHelperTest, QueryStringFiltered) {
            "https://example.com/?fbclid=&foo=1&bar=2"},
           {"http://u:p@example.com/path/file.html?foo=1&fbclid=abcd#fragment",
            "http://u:p@example.com/path/file.html?foo=1#fragment"},
+          {"https://example.com/?__s=1234-abcd", "https://example.com/"},
```

Comment:
mind extending the test to verify that shields off allow everything?

Reply:
Added a browsertest for that.